Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentConsumer/IncidentConsumer.csproj

```diff
@@ -10,13 +10,17 @@
         <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.3.0" />
```

Comment:
You probably don't need this obsolete library because you switched to a new one, i.e. Microsoft.Identity.Client

Reply:
fixed